Amazon/Pay/API/HttpCurl.php

```diff
@@ -85,8 +81,6 @@ private function execute($ch)
     public function invokeCurl($method, $url, $payload, $postSignedHeaders)
     {
         $curtime = microtime(true);
-        $response = array();
```

Comment:
If the try gives an error response is not setted, please leave $response declaration